### ISSUE TITLE
Create collaboration guidelines for Ambassadors for Design

### ISSUE THREAD

**Issue by isabelcosta** [anitab-org/documentation#30]
## Description

@annabauza has clear guidelines for designer collaboration. It would be great if this could be in a public document we can reference to ambassadors while being transparent with the whole community.

## Suggestions

This could be a document `design.md` in the root of the project, or a `design` folder with a document only dedicated to collaboration guidelines

**Comment by annabauza** [comment]
**How to contribute in designs:**
	→ It is expected that 1 designer is contributing to selected project.
	→ Join a project session to check what is needed to design, familiarise yourself with metrics guidelines if exists.
	→ Before starting the design make sure you contact the primary designer for the project to coordinate work which has NOT been assigned yet.
	→ Start design and attend design sessions regularly to get feedback.
	→ Once the designs are good, create issues for the developer. This issue need to be approved by the design team member and has to contain all necessary informations including margins, padding, accessibility, font metrics, colors, behavior on size change, described expected animations, etc.
	→ If the project does not have a metrics document - this should be created with first design.
	→ Feel free to review other designers' work either on design sessions or github issues and rise your concerts of inconstancy related to other designs within the project or related to metrics document. Feel free to review current state and rise design bugs if you find any.

**Comment by isabelcosta** [comment]
I was mentioning about what you wrote in the Membership SOW (WIP) document about the membership levels:
```
Collaboration Guidelines

- Follow consistent good designs practices:
    - AnitaB.org design colors, patterns and fonts
    - Follow known good design patterns and practices (eg. metrics.io)
    - Follow metrics guideline for your project
    - Make sure there is consistency between the platforms when it does not interfere with platform expectations (for example search on Android and iOS will look different, but should appear in the same user journey point and should use the same color palette and fonts)
(...)
```
I can create a PR with it, I was leaving up to you for you to be the author of that content :)

**Comment by annabauza** [comment]
Leave it with me @isabelcosta :)

**Comment by annabauza** [comment]
@isabelcosta what do you think - https://github.com/anitab-org/stem-diverse-tv-cms/wiki/Design-Contributions

**Comment by isabelcosta** [comment]
That looks good @annabauza 👏 I trust you with these guidelines, I just wanted to make sure there are design guidelines that are community-wide standards available to everyone in this repository regardless of the project. Since there are projects which don't have a specific ambassador yet. We could also go more specific with certain guidelines in each project's wiki, just like you did.

**Comment by annabauza** [comment]
Yep that's the idea. I'll add those to other projects. Thanks @isabelcosta

**Comment by isabelcosta** [comment]
That sounds great 💪 @annabauza
